README.md

```diff
@@ -197,23 +197,27 @@ Any value in the configuration language can be a variable reference (`key=$VALUE
 
 ## Clustering
 
-NATS supports running each server in clustered mode. You can cluster servers together for high volume messaging systems and resiliency and high availability. Clients are cluster-aware. See also the [clustered NATS](http://nats.io/documentation/server/gnatsd-cluster/) documentation.
+Clustering lets you scale NATS messaging by having multiple NATS servers communicate with each other. Clustering lets messages published to one server be routed and received by a subscriber on another server. See also the [clustered NATS](http://nats.io/documentation/server/gnatsd-cluster/) documentation.
 
-The following cluster options are supported:
+### Full mesh recommended
+
+NATS servers have a forwarding limit of one hop. This means that each `gnatsd` instance will **only** forward messages that it has received **from a client** to the immediately adjacent `gnatsd` instances to which it has routes. Messages received **from** a route will only be distributed to local clients. Therefore a full mesh cluster, or complete graph, is recommended for NATS to function as intended and as described here. 
```

Comment:
to all connected servers that represent interest in the message's published subject. Again not recommended, required, otherwise cluster is broke.